app/domain/errors.rb

```diff
@@ -558,6 +558,18 @@ module AuthnJwt
              "The valid format should meet the following regex: '{1-claim-path-format}'",
         code: "CONJ00116E"
       )
+
+      InvalidTokenAppPropertyClaimPath = ::Util::TrackableErrorClass.new(
+        msg: "Failed to parse `token-app-property` claim path: '{0-claim-path}'. "\
+             "The claim path is in an invalid format. " \
+             "The valid format should meet the following regex: '{1-claim-path-format}'",
+        code: "CONJ00117E"
+      )
+
+      TokenAppPropertyValueIsNotString = ::Util::TrackableErrorClass.new(
```

Comment:
Do we have any idea why this was failed?
does the error explain the reason?

Reply:
> Authentication Error: #<Errors::Authentication::AuthnJwt::InvalidTokenAppPropertyValue: CONJ00117E Failed to parse 'token-app-property' value. Error: '#<Errors::Authentication::AuthnJwt::InvalidClaimPath: CONJ00116E Failed to parse claim path: 'account[0]/project/id'. The claim path is in an invalid format. The valid format should meet the following regex: '(?-mix:^[a-zA-Z|$|_][a-zA-Z|$|_|0-9|.]*(\/[a-zA-Z|$|_][a-zA-Z|$|_|0-9|.]*)*$)'>'>

This is how it will look like, it is clear, thanks @sashaCher